met-api/src/met_api/resources/engagement_members.py

```diff
@@ -60,3 +60,20 @@ def post(engagement_id):
             return MembershipSchema().dump(member), HTTPStatus.OK
         except BusinessException as err:
             return {'message': err.error}, err.status_code
+
+
+@cors_preflight('GET,OPTIONS')
+@API.route('/<user_id>')
+class EngagementMembershipUser(Resource):
+    """Resource for fetching memberships for user."""
+
+    @staticmethod
+    @cross_origin(origins=allowedorigins())
+    @_jwt.has_one_of_roles([Role.VIEW_ASSIGNED_ENGAGEMENTS_SELF.value])
+    def get(engagement_id, user_id):  # pylint: disable=unused-argument
```

Comment:
i think we don't need any specfic role for this since it gives the user just his assigned engagements rite?
anyone can call this and we will return None if they dont have any engagements?

Reply:
Understood, good point. Removed